inference-engine/tests/ie_test_utils/functional_test_utils/include/functional_test_utils/layer_test_utils/summary.hpp

```diff
@@ -88,12 +88,16 @@ class Summary {
 
     std::map<ngraph::NodeTypeInfo, PassRate> getOPsStats() { return opsStats; }
 
-    void updateOPsStats(const std::shared_ptr<ngraph::Function> &function, const PassRate::Statuses &status);
+    void updateOPsStats(const std::shared_ptr<ngraph::Function> &function, const PassRate::Statuses &status,
+                        const char* className);
 
-    void updateOPsStats(const ngraph::NodeTypeInfo &op, const PassRate::Statuses &status);
+    void updateOPsStats(const ngraph::NodeTypeInfo &op, const PassRate::Statuses &status, const char* className);
 
     static Summary &getInstance();
 
+    void savePartReport(const char* className, const char* opName, unsigned long passed, unsigned long failed,
```

Comment:
is it necessary to have a class name as an arguments of `updateOPsStats`? I think it was implemented for debug

Reply:
Useful for future debugging, I propose to leave it, put the default value, transfer is not required

Reply:
what is `Part`?

Reply:
Report containing detailed information about each counter change within the general report

Reply:
Of course, saveDebugReport better for this